core/templates/components/forms/schema-based-editors/schema-based-custom-editor.component.ts

```diff
@@ -90,7 +90,21 @@ implements ControlValueAccessor, Validator, AfterViewInit {
   }
 
   ngAfterViewInit(): void {
-    this.hybridForm.statusChanges?.subscribe(() => {
+    // The 'statusChanges' property is an Observable that emits an event every
+    // time the status of the control changes. The NgForm class, which our
+    // component is using, initializes 'this.form' (which is an instance of
+    // FormGroup) in its constructor. Since FormGroup extends AbstractControl
+    // (and indirectly AbstractControlDirective), it also has the
+    // 'statusChanges' property. The 'control' getter in NgForm is overridden to
+    //  return 'this.form'.Thus, whenever we reference 'statusChanges' in our
+    // component, it is referring to 'statusChanges' of 'this.form'.Because
```

Comment:
Nit: Add space before Because

Reply:
Also maybe add paragraph break before "Because" since it's a new line.

Reply:
Done (both of them)